lib/playground.dart

```diff
@@ -456,6 +456,8 @@ class Playground implements GistContainer, GistController {
   void _initLayout() {
     editorPanelHeader = DElement(_editorPanelHeader);
     editorPanelFooter = DElement(_editorPanelFooter);
+    var editor = querySelector('#editor-host');
+    editor.setAttribute('style', 'margin:0;');
```

Comment:
I think this should use CSS - the CSS styles  for this element are here: https://github.com/dart-lang/dart-pad/blob/master/web/styles/styles.scss#L148

Reply:
Sorry, didn't know where the Styles were. Thanks, I have updated the branch.